processor/error/schema.go

```diff
@@ -380,7 +380,7 @@ var errorSchema = `{
                     "description": "The record severity.",
                     "type": ["string", "null"],
                     "default": "error",
-                    "enum": ["debug", "info", "warning", "error", "fatal"],
+                    "enum": ["debug", "info", "warning", "error", "fatal", null],
```

Comment:
is this one necessary, if type is already `string, null` ?

Reply:
yes unfortunately it is.